opentsdb-ingestion.md

```diff
@@ -0,0 +1,160 @@
+## OpenTSDB Ingestion
+
+There are 2 methods for ingesting OpenTSDB data into IRONdb:
+
+1. RESTful HTTP POST of OpenTSDB JSON formatted datapoint(s)
+2. Network socket listener akin to the normal OpenTSDB telnet method
+
+For the HTTP method, POST a JSON object (or an array of JSON objects) to
+the RESTful API endpoint (see the section below - *Writing OpenTSDB with HTTP*).
+Each datapoint should be encoded as follows:
+
+```
+{
+  "metric": "metric_name",
+  "timestamp": timestamp,
+  "value": value,
+  "tags": {
+    "tag_key": "tag_value",
+    "tag_key2": "tag_value2",
+    ...
+    "tag_keyn": "tag_valuen"
+  }  
+}
+```
+
+At least one tag key/value pair is required. Multiple datapoints can be sent
+as a JSON array of the datapoints, separated by commas and the entire POST
```

Comment:
I would just say, "as a JSON array", eliding "of the datapoints" which is redundant.

Reply:
done